.github/workflows/ft_tests.yml

```diff
@@ -0,0 +1,48 @@
+name: Free Threading Tests
+on: [push, pull_request]
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install uv and set the python version
+        uses: astral-sh/setup-uv@v5
+        with:
+          version: "0.7.2"
+          python-version: 3.13t
+          enable-cache: false
+      - name: Set up Python
+        run: uv python install
+      - name: Install Dependencies
+        run: |
+          set -x
+          set -e
+          uv pip install packaging setuptools \
+          pyflakes pytest-cov pytest-flakes \
```

Comment:
Is there a linter in this repo that forces making these lines so short? It's hard to read. Best on one line, or otherwise I suggest indenting the continuation lines so it's clear what lines go together.